src/components/Calculator.js

```diff
@@ -1,70 +1,69 @@
-import React from 'react';
 import './Calculator.css';
+import { useState } from 'react';
 import calculate from '../logic/calculate.js';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.handleClickEvent = this.handleClickEvent.bind(this);
-  }
-
-  handleClickEvent(event) {
+function Calculator() {
+//   constructor(props) {
+//     super(props);
+//     this.state = {
+//       total: null,
+//       next: null,
+//       operation: null,
+//     };
+//     this.handleClickEvent = this.handleClickEvent.bind(this);
+//   }
```

Comment:
- [ ] Kindly make sure there are no left commented codes in your codebase as it's not a good practice for clean code.